lib/wrapbox.rb

```diff
@@ -25,7 +31,23 @@ def run_cmd(*args, runner: nil, config_name: nil, **options)
       config = @configs.get(config_name)
       config.run_cmd(*args, **options)
     end
+
+    def log_level
```

Comment:
[imo] I think this method is unnecessary or we should define `log_level=`.
Currently `Wrapbox.log_level = :debug` won't work because `Wrapbox.logger.level` is set on the module definition. We have to set the log level directly like `Wrapbox.logger.level = :debug`.

[imo] I think it would be better if we can specify the log level via command line arguments (e.g. `--log-lelvel=`, `-v`, `-vv`.) or a configuration file.

Reply:
I see. It’s reasonable.